test/gerkin/tests/features/db_delete.feature

```diff
@@ -0,0 +1,14 @@
+Feature: API endpoint that allows user to delete database schema.
+  Request endpoint: DELETE /database/{id}/
+
+  Background:
+    Given database with the valid schema has been set up
+
+  Scenario: Successfully delete database schema
+    When I make a DELETE request with a valid payload
+    Then I receive a HTTP 200 response
```

Comment:
Type of response is irrelevant for all /database/ test cases. Please focus on second part which is actual information relevant for an actor making given request. Example: 
Instead of 
```gherkin
Then I receive a HTTP 200 response
And I receive a "Success" message
```
add something like: 
```gherkin
Then database was successfully removed. 
And I receive information about success of operation.
```
Same is the case for other requests.